packages/More/src/index.tsx

```diff
@@ -8,13 +8,15 @@ interface IMoreProps {
   displayBg?: boolean;
   children?: any;
   onClose: () => void;
+  disabled?: boolean;
```

Comment:
remember adding the new prop to https://github.com/thebyte9/blaze-components-react/blob/dev/packages/More/README.md

Reply:
👍